docs: improve readme (#383)
redhat-appstudio/e2e-tests: README.md

2. Run the e2e tests:
The `e2e-appstudio` command is the root command that executes all test functionality. To obtain all available flags for the binary please use `--help` flags. All ginkgo flags and go tests are available in `e2e-appstudio` binary.

Some tests could require you to have specific container image repo's created (if you're using your own container image org/user account (`QUAY_E2E_ORGANIZATION`) or your own GitHub organization (`MY_GITHUB_ORG`) 
In that case, before you run the test, make sure you have created
* `test-images` repo in quay.io, i.e. `quay.io/<QUAY_E2E_ORGANIZATION>/test-images` and make it **public**

>   * also make sure that the docker config credentials (encoded in the value of `QUAY_TOKEN` environment variable) contain a correct credentials (with proper permissions) that will be used for pushing images to that repo

Reviewer: Do we need this note as well: `make sure the robot account you're using to push to test-images repo has the required permissions set for that "test-images" repo.`? or it's too obvious?

Author: thanks, I rephrased that, but not sure if it didn't end up being totally confusing 🙈 please check and in case it's a disaster, please add a suggestion.

>   * also make sure that the docker config, that is encoded in the value of `QUAY_TOKEN` environment variable, contains a correct credentials required to push to `test-images` repo. And make sure the robot account or user account, that will be used to push to `test-images` repo in tests, has the **write** permissions set for that repo

Reviewer: ```suggestion
  * also make sure that the docker config, that is encoded in the value of `QUAY_TOKEN` environment variable, contains a correct credentials required to push to `test-images` repo. And make sure the robot account or user account has the **write** permissions set for `test-images` repo which is required by the tests to push the generated artifacts.
```
totally up to you Pavel, feel free to modify or ignore this suggestion.
